content/blog/2024-09-10-memory-access-amortization.md

```diff
@@ -0,0 +1,158 @@
++++
+# `title` is how your post will be listed and what will appear at the top of the post
+title= "Memory Access Amortization in Valkey "
+# `date` is when your post will be published.
+# For the most part, you can leave this as the day you _started_ the post.
+# The maintainers will update this value before publishing
+# The time is generally irrelvant in how Valkey published, so '01:01:01' is a good placeholder
+date= 2024-09-11 01:01:01
+# 'description' is what is shown as a snippet/summary in various contexts.
+# You can make this the first few lines of the post or (better) a hook for readers.
+# Aim for 2 short sentences.
+description= "Improve performance by enabling the processor's speculative execution capability"
+# 'authors' are the folks who wrote or contributed to the post.
+# Each author corresponds to a biography file (more info later in this document)
+authors= [ "dantouitou", "uriyagelnik"]
++++
+<style>
+  .inverted-code {
+    font-family: "Courier New", Courier, monospace;
+    background-color: #333; /* Dark background */
+    color: #f0f0f0;         /* Light text */
+    padding: 10px;
+    border-radius: 5px;
+    display: block;
+    white-space: pre-wrap;
+  }
+</style>
```

Comment:
We probably want this to be in the general _valkey.scss and have it apply to other code blocks. I don't know a lot about this though, maybe Kyle does.

Reply:
No styles are needed. You can use triple backticks to fence off the code and it will do syntax highlight automatically (see below).